src/Components/SearchBar.js

```diff
@@ -9,6 +76,7 @@ function SearchBar() {
           id="search-input"
           name="search-input"
           placeholder="Buscar Receita"
+          onChange={ (target) => handleChange(target, setSearchWord) }
```

Comment:
Essa parte do código funciona muito bem, e está correta.
Porém, a variavel target não parece estar nomeada corretamente, uma vez que ela se refere a event, e não target.
Além disso, você poderia reescrever ele de forma a evitar um, dos dois processos feitos, se utilizasse essa sintaxe:
```suggestion
          onChange={ ({ target }) => setSearchWord(target.value)) }
```

Reply:
Boa demais. Fiz a alteração seguindo sua sugestão e temos menos linhas de código.